app/Kconfig

```diff
@@ -9,7 +9,21 @@ config TWPM_USE_UNSAFE_RNG
 config TWPM_USE_HWINFO
 	bool "Use hwinfo to generate TPM unique"
 	default y
-	depends on HWINFO
+	depends on HWINFO && !TWPM_CONST_UNIQUE
+
+config TWPM_CONST_UNIQUE
+	bool "Set constant unique value"
+	help
+	  DANGER! This is intended mostly for testing or for platform bringup.
```

Comment:
Do we have issue on our problems with providing a reliable unique value?
Maybe linking this issue here would also help to understand why we are doing this.

Reply:
It's described in https://github.com/Dasharo/twpm-docs/issues/25. I updated commit message to point to my [comment](https://github.com/Dasharo/twpm-docs/issues/25#issuecomment-1997867163).